### PR TITLE
Remove old ledger for evaluator and import validator utility.

### PR DESCRIPTION
**Pull request by winder**
## Summary

The old postgres ledger is only used by a utility that we don't run. This PR removes the `import-validator` utility and deletes the old code.

### CODE REVIEW

**Review by tzaffi — COMMENTED** [review]
Seems incontrovertible to me. However, Circle is failing on [postgres_rand_test.go](https://app.circleci.com/pipelines/github/algorand/indexer/1632/workflows/4a3a3721-d1fa-446e-be11-ef58b05a7173/jobs/2794?invite=true#step-113-572) which is exactly the test which I've be wrestling with.

**Comment by winder** [comment]
Replaced by #1146

File: idb/postgres/postgres_rand_test.go [hunk 1]
```diff
@@ -2,25 +2,28 @@ package postgres
...
 	"github.com/algorand/indexer/util/test"
-	"github.com/jackc/pgx/v4"
-	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
 )
 
 func generateAddress(t *testing.T) basics.Address {
 	var res basics.Address
-
+	rand.Seed(1234)
```
Comment (winder) [inline idb/postgres/postgres_rand_test.go:26]: This isn't needed, I don't remember why I added it.

**Review by winder — COMMENTED** [review]
LGTM

**Pull request closed without merging**